### DISCUSSION TITLE
Disable automatic build

### DISCUSSION THREAD

**Discussion by minhduc0711** [mfussenegger/nvim-jdtls#119] [Q&A] [answered]
I saw that the vscode-java extension has the option to disable automatic build upon saving a file (https://github.com/redhat-developer/vscode-java/issues/406#issuecomment-361950242).
Is something similar available in `nvim-jdtls`?

**Comment by mfussenegger** [comment, marked as answer]
Yes. The option is the same. It is for the server, not for the client. You have to add it to the `settings` part of your `config`:

```lua
config = {...}
config.settings = {
  java = {
    autobuild = {
      enabled = false
    }
  }
}
```

**Reply by minhduc0711** [reply]
Thank you, I should've noticed that the related PR was created in the jdtls repo :raised_hands:
